rust/perspective-viewer/build.js

```diff
@@ -80,12 +80,13 @@ async function build_all() {
     execSync(`cargo bundle ${IS_DEBUG ? "" : "--release"}`, INHERIT);
 
     // JavaScript
+    const { default: cpy } = await cpy_mod;
+    await cpy("package.json", "src/ts");
```

Comment:
Don't copy files within the source tree (see above notes on `/build` and `/dist`), but also _especially_ don't do this with the `package.json`, which breaks node's module resolution in this directory (or any package root metadata like `Cargo.toml`).